larpandoracontent/LArHelpers/LArInteractionTypeHelper.h

```diff
@@ -198,10 +235,11 @@ class LArInteractionTypeHelper
      *
      *  @param  nuanceCode the nuance code
      *  @param  mcPrimaryList the list of primary mc particles
+     *  @param  testBeamHierarchyMode
```

Comment:
We'll have to see if we can come up with some neat text to describe exactly what's different under this mode.